package.json

```diff
@@ -177,6 +181,8 @@
     "test": "NODE_PATH=node_modules jest --config=jest.config.js --no-cache",
     "test:watch": "npm test -- --watch",
     "testing": "NODE_ENV=testing webpack-dev-server --config ./webpack/webpack.dev.js --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
+    "tsc": "NODE_ENV=development $(npm bin)/tsc",
```

Comment:
I think we don't need that, right?

Reply:
That one maybe not, but the watch is actually useful. I think for consistency if we have `tsc:watch` we should have `tsc`.

Reply:
For me it is a little source of confusion, I would think I need to run tsc now besides every npm start

Reply:
If you want to use it locally with the watch, you could just do `npx tsc -w --noEmit` 👍